simplified-viewer-epub-readium1/src/main/assets/host_app_feedback.js

```diff
@@ -82,6 +83,10 @@ $(document).ready(function()
         {
           window.location.href = "readium:media-overlay-tts-stop";
         };
+
+        this.onContentDocumentLoaded = function() {
+          window.location.href = "readium:content-document-loaded";
```

Comment:
Following the existing pattern: JavaScript events from Readium are bridged to Java by using the WebView's location as a message bus.